tools/mapmerge2/requirements.txt

```diff
@@ -1,3 +1,3 @@
-pygit2>=0.27.2
-bidict>=0.13.1
-Pillow>=5.1.0
+pygit2==1.0.1
+bidict==0.13.1
+Pillow==7.0.0
```

Comment:
What happens if you have higher versions installed?

Reply:
Good question apparently without a direct answer, just based on a quick search. I'll get back to you

Reply:
The py script installs and uses the specific version listed in requirements.txt, and pip is responsible for keeping the specific version separate from whatever you've got installed.